### PR TITLE
rename multilab measures (PLEASE REVIEW)

### PR DESCRIPTION
**Pull request by berndbischl**
hi, this is a branch which must be merged ASAP

some new measure names for multilab where either named inconsistently or weirdly.

you are free to help me to spot what i missed, and can comment here, but DO NOT CLICK merge.

### CODE REVIEW

File: R/measures.R [hunk 1]
```diff
@@ -767,34 +767,34 @@ measureHAMLOSS = function(truth, response) {
   mean(truth != response)
 }
 
-#' @export subset01
+#' @export mlsubset01
 #' @rdname measures
 #' @format none
 subset01 = makeMeasure(id = "subset01", minimize = TRUE, best = 0, worst = 1,
```
Comment (schiffner) [inline R/measures.R]: must be `mlsubset01`

File: R/measures.R [hunk 2]
```diff
@@ -767,34 +767,34 @@ measureHAMLOSS = function(truth, response) {
...
   properties = c("multilabel", "req.pred", "req.truth"),
   name = "Subset-0-1 loss",
-  note = "Proportion of observations whose classes are not completely correctly predicted, 
+  note = "Proportion of observations whose classes are not completely correctly predicted,
   following the definition by Charte and Charte: https://journal.r-project.org/archive/2015-2/charte-charte.pdf",
   fun = function(task, model, pred, feats, extra.args) {
-    measureSUBSET01(getPredictionTruth.PredictionMultilabel(pred),
+    measureSubset01(getPredictionTruth.PredictionMultilabel(pred),
```
Comment (schiffner) [inline R/measures.R]: Isn't our convention that the second part of the function name is uppercase? (Like measureMMCE etc.)?

File: R/measures.R [hunk 3]
```diff
@@ -767,34 +767,34 @@ measureHAMLOSS = function(truth, response) {
...
+#' @export measureSubset01
 #' @rdname measures
 #' @format none
-measureSUBSET01 = function(truth, response) {
+measureSubset01 = function(truth, response) {
   mean(!apply(truth == response, 1, all))
 }
 
-#' @export f1mult
+#' @export f1Multilab
```
Comment (schiffner) [inline R/measures.R]: When I compare with NEWS this might be `mlf1` instead

File: R/measures.R [hunk 4]
```diff
@@ -805,7 +805,7 @@ f1mult = makeMeasure(id = "f1mult", minimize = FALSE, best = 1, worst = 0,
 #' @export measureF1MULT
 #' @rdname measures
 #' @format none
-measureF1MULT = function(truth, response) {
+measureF1MultiLab = function(truth, response) {
```
Comment (schiffner) [inline R/measures.R]: Probably `measureMLF1`

File: R/measures.R [hunk 5]
```diff
@@ -814,27 +814,27 @@ measureF1MULT = function(truth, response) {
       fi[i] = 2 * sum(truth[i, ] * response[i, ]) / (sum(truth[i, ]) + sum(response[i, ]))
     }
   }
-  mean(fi) 
+  mean(fi)
 }
 
 #' @export accmult
```
Comment (schiffner) [inline R/measures.R]: must be the same as l. 823

File: R/measures.R [hunk 6]
```diff
@@ -849,21 +849,21 @@ measureACCMULT = function(truth, response) {
 #' @export precmult
 #' @rdname measures
 #' @format none
-precmult = makeMeasure(id = "precmult", minimize = FALSE, best = 1, worst = 0,
+ppvmultilab = makeMeasure(id = "precmult", minimize = FALSE, best = 1, worst = 0,
   properties = c("multilabel", "req.pred", "req.truth"),
-  name = "Precision (multilabel)",
-  note = "Mean of ratio of truly predicted labels for each instance, 
+  name = "Postive predicted value (multilabel)",
```
Comment (schiffner) [inline R/measures.R]: I believe it's positive predictIVE value

File: NEWS [hunk 7]
```diff
@@ -76,13 +75,18 @@ mlr_2.9:
...
+-- makeMultilabelStackingWrapper
 
 - removed functions
 -- getParamSet generic (now in ParamHelpers package)
 
 - new measures
--- measureBrier (for mutliclass problems)
+-- measureBrier (for multiclass problems)
 -- logloss
+-- multilabel: mlsubset01, mlf1, mlacc, mlprec, mlrecall
```
Comment (schiffner) [inline NEWS]: names are not consistent with those in R/measures.R

**Comment by larskotthoff** [comment]
Wait, must be merged ASAP and DO NOT MERGE? I'm confused.

**Comment by berndbischl** [comment]
> Wait, must be merged ASAP and DO NOT MERGE? I'm confused.

i means i should finish it now. i had to stop coding due to teaching and wanted to have it in a PR. so "do not merge" was for you while "finish ASAP" was targeted at myself ;-)

> Isn't our convention that the second part of the function name is uppercase? (Like measureMMCE etc.)?

no it really is not! we did that for cases where the second word was an "abbreviation" like MMCE CMAES and so on. bit not for "full" words. (actually this conventions is quite annoying because of borderline cases, but thats how most functions are named in mlr now)

**Comment by berndbischl** [comment]
this should really be done now. please review, wanna merge asap

**Comment by ja-thomas** [comment]
Just looked over this, seems good to me, the new names seem to be consistent. But maybe someone with a bit more mlr experience than me should  have a look as well.

**Pull request merged**